src/main/webapp/app/pages/curation/collapsible/Collapsible.tsx

```diff
@@ -83,14 +85,23 @@ export default function Collapsible({
         )}
         ref={node => {
           if (node && borderLeftColor) {
+            const hadBackgroundColor = backgroundColor;
             if (disableCollapsible) {
               borderLeftColor = DISABLED_NEST_LEVEL_COLOR;
+              if (!hadBackgroundColor) {
+                backgroundColor = DISABLED_NEST_LEVEL_COLOR;
```

Comment:
Do we need `hadBackgroundColor`? Can we just change to `if (!backgroundColor)`?

Reply:
Mainly used in:
```
getHexColorWithAlpha(backgroundColor, hadBackgroundColor ? 0 : 0.05)
```
By default, the background color = border left color with 0.05 opacity. But if background color was passed as prop, then it should not have opacity.